config/requirements_docker.txt

```diff
@@ -0,0 +1,8 @@
+-r requirements_core.txt
+
+# Serving
+honcho==1.0.1                           # Allows a Procfile to run multiple processes
+gunicorn==20.0.4                        # Primary server for wsgi
+
+# Cache
+django-redis==4.12.1                    # Use redis for cache (on heroku; local optional)
```

Comment:
If Redis itself is in core, might make more sense to include it there (and remove the comment).